text/2019-08-21-supported-platforms.md

```diff
@@ -0,0 +1,230 @@
+        or three regular contributors
+
+### Tier Definitions
+
+This RFC proposes the following definitions for the tiers of platforms and targets:
+
+* **Deployment Platforms**
+      + **Tier 1**:
+        - CentOS 7.x.x on x86-64 on x84-64 with SSE4.2
+        - Ubuntu 16.04 through 19.04 on x84-64 with SSE4.2
+      + **Tier 2**:
+        - Any Linux Kernel >3.x with glibc >2.17 on x86-64 with SSE4.2
+        - Any Linux Kernel >3.x with glibc >2.17 on ARM64/AARCH64
+      + **Tier 3**:
+        - Any Linux Kernel >3.x with glibc >2.17 on x86-64 without SSE4.2
+        - Any Linux Kernel >3.x with musl libc on X86-64 or ARM64/AARCH64, with
+          or without SSE4.2
+        - The latest Mac OS X release on X86-64 (Caution: Apple offers no
+          written support cycle)
+      + **Not supported**:
+        - Windows
+        - FreeBSD
+        - Any Linux using the 'Unbreakable Enterprise Kernel' by Oracle.
+* **Development Platforms**
+      + **Tier 1**:
+        - Any Linux kernel >3.x on X84-64 with glibc 2.17 with SSE4.2 or without
+        - The latest Mac OS X release on X86-64
+      + **Tier 2**:
+        - Any machine with access to a Docker installation which can support
+          Linux containers
+      + **Tier 3**:
+        - Nix (the package manager) environments
+        - Any Linux Kernel >3.x with musl libc on X86-64 or ARM64/AARCH64 with
+          SSE4.2 or without
+      + **Not supported**:
+        - Windows
+        - FreeBSD
+        - Any Linux using the 'Unbreakable Enterprise Kernel' by Oracle.
+
+This list does not attempt to list all posibilities, it lists all major targets
+we are asked about on a regular basis and special cases we are aware of. If
+something is not listed, it should be considered untested and in the "Not
+supported" tier.
+
+### RFC Effects
+
+If accepted, this RFC would:
+
+* **discourage** adding this data to the TiKV documentation.
+      - This should be a living development reference for our contributors and
+        community. We should not encourage new or end users to try unsupported
+        platforms, we'll limit our messaging to only Tier 1 platforms.
+      - These targets and how we support them may change in the future, and
+        maintaining separate lists is risks out of date data.
+* **encourage** stating and listing our tier 1 supported targets where
+  appropriate in documentation, as these specifics are of great importance to
+  our readers.
+* **encourage** any future changes be ammended to this RFC itself.
+
+This RFC as currently defined does not change any of our *current* support, it
+only clarifies them and defines a path forward for new targets.
+
+## Drawbacks
+
+* We must maintain this compatability specification and ensure correctness
+* Contributors may feel encouraged to try to promote certain platforms up a tier
+  and require us to do more testing on releases.
+* This list does not talk about hardware recommendations like CPUs and memory
```

Comment:
It does mention x86 and ARM, and we might want to say that bigendian archs will never be supported

Reply:
I'll add it to the list of not supported. :)